patches/third_party-crashpad-crashpad-util-mach-mig_fix.py.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/third_party/crashpad/crashpad/util/mach/mig_fix.py b/third_party/crashpad/crashpad/util/mach/mig_fix.py
+index 037746fac3bb8eda68a04ba0af5707e7025e547f..083a69bd6144f20622c888e85999ccba540d527d 100755
+--- a/third_party/crashpad/crashpad/util/mach/mig_fix.py
++++ b/third_party/crashpad/crashpad/util/mach/mig_fix.py
+@@ -23,6 +23,13 @@ import sys
+ from mig_gen import MigInterface
+ 
+ 
++def _make_generated_comments_deterministic(contents):
```

Comment:
this is definitely not the right way to patch python files, the same thing applies as in any other file. You should define the method in a different py file and import and call it in the same line. See `mojo/public/tools/mojom/mojom_parser.py`

Reply:
unless this is an upstream change you are pulling in?

Reply:
> unless this is an upstream change you are pulling in?

yep, I mentioned this in the PR description: patches will be upstreamed, so I didn't optimise them too much to lower conflicts.
I'll sync up patches with the upstream.

Reply:
Patches are synced with upstream.